### ISSUE TITLE
Sort order in gear incorrect

### ISSUE THREAD

**Issue by tmazuryk** [Cores-ts/Fundingbox-support#201]
![zrzut ekranu 2017-08-02 o 21 40 22](https://user-images.githubusercontent.com/16816225/28891597-5546effc-77cb-11e7-8972-855824f83ea1.png)

**Comment by josealonsogarcia** [comment]
Yes, is sorting first on capital letters

**Comment by tmazuryk** [comment]
Yes, but it seems that sort order now is: ABC......Zabc.......
and it should be: AaBcCc.....

**Comment by josealonsogarcia** [comment]
Of course, I didn't express myself correctly in my last message. It's a bug and the sort order should be as you say :)
